drivers/bridge/setup_ip_tables.go

```diff
@@ -189,41 +249,50 @@ func setupIPTablesInternal(hostIP net.IP, bridgeIface string, addr net.Addr, icc
 	natRule := iptRule{table: iptables.Nat, chain: "POSTROUTING", preArgs: []string{"-t", "nat"}, args: natArgs}
 	hpNatRule := iptRule{table: iptables.Nat, chain: "POSTROUTING", preArgs: []string{"-t", "nat"}, args: hpNatArgs}
 
+	ipVersion := iptables.IPv4
+
+	if strings.Contains(address, ":") {
+		ipVersion = iptables.IPv6
```

Comment:
this should work, but should we consider using something from the standard `net` package instead ?

Reply:
Yeah I have also thought that this is not perfect but I am not 100% sure how the best way would be here in go.

Based on [this](https://stackoverflow.com/questions/22751035/golang-distinguish-ipv4-ipv6) the net package seems not be accurate in all cases.

Do you think it is enough for our needs here to simply relay on ´To4()`?

Reply:
I have seen that ´To4()` is already used in many situations so I will replace the string compare.

Reply:
replaced with `net.IP.To4()` check